index.html

```diff
@@ -0,0 +1,195 @@
+               <div class ="image-placeholder"></div>
+               <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+               <ul class ="lang">
+                   <li class="language">html</li>
+                   <li class="language">css</li>
+                   <li class="language">javascript</li>
+                   <li class="language">Ruby on rails</li>
+               </ul>
+               <div class="butt"><button class="btn" type="button"> See Project</button></div>
+           </article> 
+       </div>
+           <div class="design-3">
+           <article class="project">
+               <div class ="image-placeholder"></div>
+               <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+               <ul class ="lang">
+                   <li class="language">html</li>
+                   <li class="language">css</li>
+                   <li class="language">javascript</li>
+                   <li class="language">Ruby on rails</li>
+               </ul>
+               <div class="butt"><button class="btn" type="button"> See Project</button></div>
+           </article>
+            <article class="project">
+               <div class ="image-placeholder"></div>
+               <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+               <ul class ="lang">
+                   <li class="language">html</li>
+                   <li class="language">css</li>
+                   <li class="language">javascript</li>
+                   <li class="language">Ruby on rails</li>
+               </ul>
+               <div class="butt"><button class="btn" type="button"> See Project</button></div>
+           </article>
+           </div>
+           </div>
+       </div>
+       </section>
+
+       <section class="aboutme-background">
+       <section class="aboutme section">
+           <div class="aboutme-container">
+           <div class="about2">
+               <h2>About me</h2>
+
+           <div class="about">
+           </div>
+           <div class="about-text">
+           I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+           don’t hestiate to contact me.
+           </div>
+           <div class="butt resume-container"><button class="btn btn-resume" type="button"> Get My Resume</button></div>
+       </div>
+           </div>  
+           
+           <div class="skills-container">
+           <article class="project2">
+           <div class="language-icon"><img src="./images/icon-languages (1).png" alt=""></div>
+           <div class="lang-text">Languages </div>
+               <ul>
+               <div class="lang-text-list">
+                   <li  class="li-lang">Javascript</li>
+                   <li class="li-lang">Ruby</li>
+                   <li class="li-lang">html</li>
+                   <li class="li-lang">css</li>
+               </div>
+               </ul>
```

Comment:
- [ ] In this section it's unnecessary to wrap the `li` tags inside a `div`, please remove this div. If you can check the [ul](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) explanation, it's used to wrap your list of items `li`